sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/azure.yaml

```diff
@@ -0,0 +1,77 @@
+provider:
+  metadata:
+    name: Azure 
+    category: Cloud provider
+    references:
+      - description: OWASP Top 10 2021 Category A7 - Identification and Authentication Failures
+        link:  https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures/
+        type: Standards
+      - description: docs.microsoft.com - Manage storage account access keys
+        link:  https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal
+        type: Documentation
+      - description: OWASP Top 10 2017 Category A3 - Sensitive Data Exposure
+        link:  https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
+        type: Standards
+      - description: MITRE, CWE-798 - Use of Hard-coded Credentials
+        link:  https://cwe.mitre.org/data/definitions/798
+        type: Standards
+      - description: MITRE, CWE-259 - Use of Hard-coded Password
+        link:  https://cwe.mitre.org/data/definitions/259
+        type: Standards
+      - description: SANS Top 25 - Porous Defenses
+        link:  https://www.sans.org/top25-software-errors/#cat3
+        type: Standards
+    message: Make sure this Azure Storage Account Key gets revoked, changed, and removed from the code.
+  detection:
+    post:
+        patternNot: "EXAMPLE"
+
+  rules:
+    - id: S6338
+      metadata:
+        name: Azure Storage Account Keys should not be disclosed
```

Comment:
As with the other PR, I feel like this is the message, not the name.

Reply:
totally 
```suggestion
        name: Azure Storage Account Keys
```